Sources/PlaybackSDK/PlayerUIView/PlaybackUIView.swift

```diff
@@ -28,23 +25,42 @@ internal struct PlaybackUIView: View {
     /// State variable to track whether video details have been fetched or not.
     @State private var hasFetchedVideoDetails = false
     
-    /// State variable to store the HLS stream URL.
-    @State private var videoURL: URL?
+    /// The fetched video details of the entryIDs
+    @State private var videoDetails: [PlaybackResponseModel]?
+    /// Array of errors for fetching playlist details
+    @State private var playlistErrors: [PlaybackAPIError]?
+    /// Error of failed API call for loading video details
+    @State private var failureError: PlaybackAPIError?
+    
+    /// Closure to handle errors during a single HLS stream loading.
+    private var onError: ((PlaybackAPIError) -> Void)?
+    /// Closure to handle multiple errors during Playlist stream loading.
+    private var onErrors: (([PlaybackAPIError]) -> Void)?
```

Comment:
> Looks like these handlers are supposed to be called in different cases, do we need to handle them differently on the screen/in the app? If we do, could you please rename them to include more information about their use cases, to make the difference clearer in the naming?

I've added comment on top of every variable. The first one `onError` returns a single error for single entryId request. The second one `onErrors` (plural) returns a list of errors because the entryIds required are more than one so accordingly the error potentially could be multiples.